lib/api/unittest/testfiles/new_quantilesState.json

```diff
@@ -1,6 +1,6 @@
+,{"mlcue":"inflb16822970713042737015","mlkey":"api","mlquantilesdescription":"airline","mlversion":"3","time":"1486855800","a":{"a":"0","b":"470","c":"","d":{"a":"201","b":"479","c":{"a":"0","b":"32767","c":"0"},"c":{"a":"0","b":"3","c":"1"},"c":{"a":"0","b":"0","c":"470"},"c":{"a":"1","b":"1","c":"3"},"c":{"a":"4","b":"7","c":"1"},"c":{"a":"8","b":"8","c":"1"},"c":{"a":"9","b":"9","c":"1"},"c":{"a":"16384","b":"20479","c":"1"},"c":{"a":"20480","b":"24575","c":"1"}},"e":{"a":"201","b":"9","c":{"a":"0","b":"32767","c":"0"},"c":{"a":"1","b":"1","c":"3"},"c":{"a":"3","b":"3","c":"1"},"c":{"a":"4","b":"4","c":"1"},"c":{"a":"8","b":"8","c":"1"},"c":{"a":"9","b":"9","c":"1"},"c":{"a":"19875","b":"19875","c":"1"},"c":{"a":"21954","b":"21954","c":"1"}},"f":"2.000000","h":"0","g":{"d":"1","a":{"a":"4976084314140307180","b":{"a":"19.64787","b":"0"}}}}}
+,{"mlcue":"sysChange","mlkey":"api","mlquantilesdescription":"root","mlversion":"3","time":"1486855800","a":{"a":"0","b":"470","c":"","d":{"a":"201","b":"479","c":{"a":"0","b":"32767","c":"0"},"c":{"a":"0","b":"3","c":"1"},"c":{"a":"0","b":"0","c":"470"},"c":{"a":"1","b":"1","c":"3"},"c":{"a":"4","b":"7","c":"1"},"c":{"a":"8","b":"8","c":"1"},"c":{"a":"9","b":"9","c":"1"},"c":{"a":"16384","b":"20479","c":"1"},"c":{"a":"20480","b":"24575","c":"1"}},"e":{"a":"201","b":"9","c":{"a":"0","b":"32767","c":"0"},"c":{"a":"1","b":"1","c":"3"},"c":{"a":"3","b":"3","c":"1"},"c":{"a":"4","b":"4","c":"1"},"c":{"a":"8","b":"8","c":"1"},"c":{"a":"9","b":"9","c":"1"},"c":{"a":"19875","b":"19875","c":"1"},"c":{"a":"21954","b":"21954","c":"1"}},"f":"1.000000","h":"0","g":{"d":"1","a":{"a":"7382284430162962205","b":{"a":"19.64787","b":"0"}}}}}
```

Comment:
It looks like this has changed `"b":"19.64787"` to `"b":{"a":"19.64787","b":"0"}`.

I think from a BWC perspective it would be nicer if it could instead be changed to `"b":"19.64787","c":"0"`.  Otherwise it will fail to parse if a job runs on a 6.5 but then on 6.4.

Reply:
This is a bigger change because we currently use the `CPersistUtils` framework to persist this container. Do we think that it is a realistic scenario that we will need to restore 6.5 state into a 6.4 analytics process because I think there will potentially be other issues around trying to load newer state into an older model.

Reply:
Turns out my worry was misplaced, because the whole `"b"` section is new in 6.5, which is not released.  So no changes necessary.